supervised/automl.py

```diff
@@ -7,6 +7,14 @@
 import pandas as pd
 import logging
 from tabulate import tabulate
+from abc import ABC
+from copy import deepcopy
+
+from varname import nameof
+
+from sklearn.base import BaseEstimator
+from sklearn.utils.validation import check_array
+from sklearn.metrics import r2_score, accuracy_score
```

Comment:
These metrics are used in `score()`. This method must return a goodness of `fit` measure (higher the better). I choose these metrics because they are the most common ones.

Reply:
OK, let's leave it as it is. When implementing user set `eval_metric` we can improve on this.